### PR TITLE
Document third party Ingress controllers

### PR DESCRIPTION
**Pull request by catherineluse**

### CODE REVIEW

File: content/k3s/latest/en/third-party-integrations/nginx-ingress/_index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: ingress-nginx
+  namespace: kube-system
+spec:
+  chart: nginx-ingress
+  repo: https://kubernetes-charts.storage.googleapis.com
+  targetNamespace: ingress-nginx
+  version: v1.30.3
```
Comment (dnoland1) [inline content/k3s/latest/en/third-party-integrations/nginx-ingress/_index.md:80]: Can bump this to `v1.39.1` to match what is used by default with latest RKE cluster.

Comment (catherineluse) [reply content/k3s/latest/en/third-party-integrations/nginx-ingress/_index.md]: updated

**Comment by catherineluse** [comment]
@davidnuzik if this PR is merged, I can go to the original PR that was adding the Ambassador information to the Networking page, and change it to just say to go to this third-party section for more ingress controller options.

**Comment by davidnuzik** [comment]
LGTM

**Comment by davidnuzik** [comment]
We're going to standby for now. We are having discussions on how we want to approach this. The 3rd party message in docs may not be the best route.

**Comment by catherineluse** [comment]
This PR won't be merged because third party integrations are going to be handled under Rancher's blog/writers' program.

**Pull request closed without merging**